adding a rest api

ciurana-life opened the pull request.

ciurana-life committed:
adding a rest api

c0dearm commented on app/blog/models.py:
```diff
@@ -25,6 +25,9 @@ def save(self, *args, **kwargs):
         return super().save(*args, **kwargs)
 
+    class Meta:
+        ordering = ["-created_at"]
```
ordering adds quite a performance overhead in all your queries. If you are sure you are always going to need this ordering, this is fine. Otherwise I would just specify it in your queries whenever you need it.

ciurana-life replied:
Pagination may yield inconsistent results with an unordered object_list

c0dearm commented on tests/blog_api/test_views.py:
```diff
@@ -0,0 +1,71 @@
+
+
+@pytest.fixture
+def factory():
```
usually, we put fixtures in a `conftest.py` file, in this case in `tests/blog_api/conftest.py` to keep them separated from the actual tests

ciurana-life committed:
Review: fixtures on own file, removed ordering from models

Left the warning open on the tests about ordering just in case it gets called with another function

c0dearm reviewed (approved):
Very nice work!
